todo-service/src/main/java/com/brtvsk/todoservice/service/TodoServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+package com.brtvsk.todoservice.service;
+
+import com.brtvsk.todoservice.exception.TodoNotFoundException;
+
+import com.brtvsk.todoservice.model.dto.OptionalRequestTodoDto;
+import com.brtvsk.todoservice.model.dto.RequestTodoDto;
+import com.brtvsk.todoservice.model.dto.ResponseTodoDto;
+import com.brtvsk.todoservice.model.entity.Todo;
+import com.brtvsk.todoservice.repository.TodoRepository;
+import com.brtvsk.todoservice.utils.TodoMapper;
+import org.springframework.stereotype.Service;
+
+import java.sql.Date;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class TodoServiceImpl implements TodoService {
+
+    private final TodoRepository todoRepository;
+
+    public TodoServiceImpl(final TodoRepository todoRepository) {
+        this.todoRepository = todoRepository;
+    }
+
+    @Override
+    public ResponseTodoDto create(final RequestTodoDto dto) {
+        Todo todo = TodoMapper.INSTANCE.fromRequestTodoDto(dto);
```

Comment:
is it possible to do `Mapper` as a bean?

Reply:
Now `TodoMapper` a bean